simple/msg_pingpong.c

```diff
@@ -40,6 +44,8 @@
 #include <rdma/fi_cm.h>
 #include <shared.h>
 
+static uint32_t FI_VER = FI_VERSION(1, 0);
```

Comment:
In that case, I would put this in a header file as a define and update the apps to reference it.  And I would use a different prefix - FT?